ntbs-service/Pages/_HomepageKpiDetails.cshtml

```diff
@@ -3,17 +3,19 @@
 <div>
     @if (@Model.HomepageKpiDetails.Any())
     {
-        <h1 class="govuk-visually-hidden"> Homepage KPI Details </h1>
+        

```

Comment:
What's the logic behind removing the `h1`? I thought it was a nice addition for SR purposes

Reply:
Looks like they got replaced by visible headings, which seems fine - though I'm not sure if we need it to be repeated in both sections?

Also I think these should be `h2`s. Best practice is to have a single `h1` on the page, so for the homepage it would be something like:
h1 NTBS Homepage
h2 KPIs
h2 Alerts
h2 Recent Drafts
h2 Recent Notifications

See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements